### DISCUSSION TITLE
Chia processes starts with NICE 10?

### DISCUSSION THREAD

**Discussion by herotm** [swar/Swar-Chia-Plot-Manager#239] [General]
Is there anyway to set priority to nice 0 for the processes

**Comment by raid5** [comment]
I've heard a lot of comments on linux process priority lately so I'm also interested in this thread.

**Comment by swar** [comment]
There will be customization added in soon. I think the defaults are fine for now, but you can feel free to change the nice_values to your specific setups. 

Change it here: https://github.com/swar/Swar-Chia-Plot-Manager/blob/943dd10a6d4b9b40474fbd891f2cb4256ca93554/plotmanager/library/utilities/jobs.py#L129-L131
